.eslintrc.json

```
@@ -1,5 +1,5 @@
 {
-  "extends": ["next/core-web-vitals", "standard", "prettier"],
+  "extends": ["next/core-web-vitals", "plugin:storybook/recommended", "standard", "prettier"],
```

`plugin:storybook/recommended`は`standard`の後だと思う
```suggestion
  "extends": ["next/core-web-vitals", "standard",  "plugin:storybook/recommended" , "prettier"],
```

---

extendsの順番を修正しました。